.github/workflows/firefox-ios-ui-tests-old.yml

```diff
@@ -0,0 +1,92 @@
+name: Focus iOS 15 & 16 tests
+on:
+  workflow_dispatch:
+
+env:
+    browser: focus-ios
+    xcode_version: 15.4
+    test_results_directory: /Users/runner/tmp
+
+jobs:
+  Focus-iOS-Tests:
+    name: Focus iOS
+    runs-on: macos-14
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - ios_version: 16.4
+            ios_simulator: iPhone 14
+          - ios_version: 15.5
+            ios_simulator: iPhone 13
+    steps:
+      - name: Check out source code
+        uses: actions/checkout@v4.1.7
+        with:
+          repository: mozilla-mobile/firefox-ios
```

Comment:
This line may not be needed because we are fetching code from this repo This option is used because I was testing using my own repo.